extensions/azurecore/package.nls.json

```diff
@@ -30,5 +30,7 @@
 	"config.azureCodeGrantMethod": "Code Grant Method",
 	"config.azureDeviceCodeMethod": "Device Code Method",
 	"config.noSystemKeychain": "Disable system keychain integration. Credentials will be stored in a flat file in the user's home directory.",
-	"config.piiLogging": "Should Personally Identifiable Information (PII) be logged in the console view locally"
+	"config.piiLogging": "Should Personally Identifiable Information (PII) be logged in the console view locally",
+	"config.loggingLevel": "[Optional] Log level for backend services. Azure Data Studio generates a file name every time it starts and if the file already exists the logs entries are appended to that file. The default tracingLevel does not log much. Changing verbosity could lead to extensive logging and disk space requirements for the logs."
```

Comment:
`backend services` doesn't really make much sense here. I'd suggest something like `The verbosity of logging for the Azure Accounts extension.` (the rest of that is also unnecessary IMO, that's already pretty clear)